lib/browser/api/web-contents.js

```diff
@@ -263,7 +263,7 @@ WebContents.prototype.takeHeapSnapshot = function (filePath) {
 }
 
 // Translate the options of printToPDF.
-WebContents.prototype.printToPDF = function (options, callback) {
+WebContents.prototype.printToPDF = function (options) {
```

Comment:
we should keep both functions until the deprecated functions are removed to ensure no regressions

Reply:
I did that